arm-batch/2015-12-01/BatchManagement.json

```diff
@@ -0,0 +1,1340 @@
+          "$ref": "#/definitions/AutoStorageBaseProperties",
+          "description": "Gets or sets properties relating to auto storage."
+        }
+      },
+      "description": "The properties of a Batch account."
+    },
+    "BatchAccountCreateParameters": {
+      "properties": {
+        "location": {
+          "type": "string",
+          "description": "The region in which the account is created."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "The user specified tags associated with the account."
+        },
+        "properties": {
+          "$ref": "#/definitions/AccountBaseProperties",
+          "description": "The properties of the account."
+        }
+      },
+      "description": "Parameters supplied to the Create operation."
+    },
+    "AutoStorageProperties": {
+      "properties": {
+        "storageAccountId": {
+          "type": "string",
+          "description": "Gets or sets the resource id of the storage account to be used for auto storage."
+        },
+        "lastKeySync": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Gets or sets the UTC time at which storage keys were last synced to the Batch account."
+        }
+      },
+      "required": [ "storageAccountId", "lastKeySync" ],
+      "description": "Contains information about the auto storage associated with a Batch account."
+    },
+    "AccountProperties": {
+      "properties": {
+        "accountEndpoint": {
+          "type": "string",
+          "description": "The endpoint used by this account to obtain Batch services."
+        },
+        "provisioningState": {
+          "type": "string",
+          "description": "The provisioned state of the resource",
+          "enum": [
+            "Invalid",
+            "Creating",
+            "Deleting",
+            "Succeeded",
+            "Failed",
+            "Cancelled"
+          ],
+          "x-ms-enum": {
+            "name": "AccountProvisioningState" 
+          }
+        },
+        "autoStorage": {
+          "$ref": "#/definitions/AutoStorageProperties",
+          "description": "The properties and status of any auto storage associated with the account."
+        },
+        "coreQuota": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The core quota for this Batch account."
+        },
+        "poolQuota": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The pool quota for this Batch account."
+        },
+        "activeJobAndJobScheduleQuota": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The active job and job schedule quota for this Batch account."
+        }
+      },
+      "description": "Account specific properties."
+    },
+    "AccountResource": {
+      "properties": {
```

Comment:
Take a look at how StorageAccount is defined. https://github.com/Azure/azure-rest-api-specs/blob/master/arm-storage/2015-06-15/swagger/storage.json#L627

Looks like Account is a child of Resource. It would be better if you follow the same pattern as Storage Account.

Do take a look at the documentation for extension "x-ms-azure-resource" https://github.com/Azure/autorest/blob/master/Documentation/swagger-extensions.md#x-ms-azure-resource.